lisa/node.py

```diff
@@ -733,6 +741,33 @@ def check_sudo_password_required(self) -> None:
                     raise RequireUserPasswordException("Reset password failed")
             self._check_password_and_store_prompt()
 
+    def _collect_logs_using_platform(self) -> Optional[str]:
+        """
+        Collects information using the RunCommand feature.
+        This is used when the connection to the node is not stable.
+        """
+        from lisa.features import RunCommand
+
+        if self.features.is_supported(RunCommand):
+            run_command = self.features[RunCommand]
+            commands = [
+                "echo 'Executing: ip addr show'",
+                "ip addr show",
+                "echo 'Executing: ip link show'",
+                "ip link show",
+                "echo 'Executing: systemctl status NetworkManager --no-pager --plain'",
+                "systemctl status NetworkManager --no-pager --plain",
```

Comment:
@squirrelsc how do we determine the command(s) to be run? The OS object is not initialized at this point